README.md

```diff
@@ -126,14 +126,22 @@ certificate-profile-name: my-profile-name
 ```
 
 ### File Specification
+#### Files List
+This strategy allows you to specify a comma separated list of files to be signed.
+
+```yaml
+# A comma separated list of absolute paths to the files being signed. Can be combined with the files-folder and file-catalog inputs.
+files: ${{ github.workspace }}\files\app.dll,${{ github.workspace }}\files\app.exe
+```
+
 #### Files Folder
 This strategy allows you to specify a folder that contains all the files you want signed. There are options available for narrowing the focus as well. For example, you can use the `files-folder-filter` input to specify that you only want `exe` files to be signed.
 
 ```yaml
 # The folder containing files to be signed. Can be combined with the file-catalog input.
 files-folder: ${{ github.workspace }}\App\App\bin\Release\net6.0-windows
 
-# A comma separated list of file extensions that determines which types of files will be signed in the folder specified by the files-folder input. Any file type not included in this list will not be signed. If this input is not used, all files in the folder will be signed.
+# A comma separated list of file extensions that determines which types of files will be signed in the folder specified by the files-folder input. Any file type not included in this list will not be signed. If this input is not used, all files in the folder will be signed. Supports wildcards for matching multiple file names with a pattern.
```

Comment:
What about the `recursive: true` scenario? Will these patterns be evaluated in all folders visited?

Reply:
Yup sounds great.